sci-biology/imagej/imagej-1.54i-r1.ebuild

```diff
@@ -65,6 +65,9 @@ src_prepare() {
 	if ! use debug ; then
 		sed -i 's: debug="on">: debug="off">:' "${S}"/build.xml || die
 	fi
+	sed -i 's/source="1\.6"/source="1.7"/g' build.xml || die
+	sed -i 's/target="1\.6"/target="1.7"/g' build.xml || die
```

Comment:
Better delete `-source` / `-target` attributes completely from alll `javac` elements of all `build.xml` files and use the `ant.build.javac.source` `ant.build.javac.target` options as done in [dev-java/c3p0](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/c3p0/c3p0-0.9.5.5-r2.ebuild#n52) or in [dev-java/lucene](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/lucene/lucene-2.4.1-r5.ebuild#n50).

Reply:
Thanks. Done.

Reply:
> Thanks. Done.

Sorry, forgot to mention, pls avoid using `jre-1.7:*` / `jdk-1.7:*`. It is no longer supported by Java 21.
https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE
Also, the support for `EAPI 7` [will be dropped soon](https://github.com/gentoo/gentoo/pull/39233) on [java-pkg-simple.eclass](https://qa-reports.gentoo.org/output/eapi-per-eclass/java-pkg-simple.eclass/7.txt).